<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>19</y>
      <width>221</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>188</width>
       <height>122</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="Url_LE"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="OnJoin_PB">
          <property name="text">
           <string>OnJoin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="UnJoin_PB">
          <property name="text">
           <string comment="UnJoin" extracomment="UnJoin">UnJoin</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="Message_LE"/>
      </item>
      <item>
       <widget class="QPushButton" name="SendMessage_PB">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QVideoRender" name="Local_W" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);</string>
    </property>
   </widget>
   <widget class="QVideoRender" name="Remote_W1" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);</string>
    </property>
   </widget>
   <widget class="QVideoRender" name="Remote_W2" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);</string>
    </property>
   </widget>
   <widget class="QVideoRender" name="Remote_W3" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVideoRender</class>
   <extends>QWidget</extends>
   <header location="global">QVideoRender.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
